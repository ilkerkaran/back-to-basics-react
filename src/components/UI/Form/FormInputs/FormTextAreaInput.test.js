import React from 'react';
import renderer from 'react-test-renderer';
import TextArea from './FormTextAreaInput';


describe('<TextArea /> snapshot ', () => {
  const inputConfig = {
    inputType: 'textArea',
    inputName: 'textArea',
    label: 'TextArea',
    isRequired: true,
    register: () => {},
    errors: {}
  };


  it('renders correctly', () => {
    const tree = renderer
      .create(<TextArea inputConfig={inputConfig}/>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
