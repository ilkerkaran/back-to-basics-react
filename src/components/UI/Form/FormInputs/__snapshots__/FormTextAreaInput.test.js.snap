// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TextArea /> snapshot  renders correctly 1`] = `
<div>
  <label
    htmlFor="textArea"
  >
    TextArea
  </label>
  <textarea
    name="textArea"
  />
  <div
    style={
      Object {
        "color": "darkred",
      }
    }
  />
</div>
`;
